### ISSUE TITLE
[ BUG ] `Uninstall-FalconSensor` fails on some 64-bit Windows machines and returns no status

### ISSUE THREAD

**Issue by dvd00** [CrowdStrike/psfalcon#315]
**Describe the bug**
The `Uninstall-FalconSensor` command fails on some 64 bit Windows machines.
Occasionally, the embded PowerShell scritp used by Uninstall-FalconSensor command fails the 32/64 bit test.
The script checks wether the returned value of `(Get-WmiObject win32_operatingsystem).osarchitecture` is `64-bit` to determine which regkey to use for retrieving the uninstaller. In some instances of Windows, this command returns `64 bit` (with a space instead of a hyphen).

**To Reproduce**
I have no clue under which conditions this bug occurs since I have observed it on different scenarios:
- OS Version: Windows 10, Windows 11, Windows Server 2019
- OS Build: 17763, 18362, 18363, 19041, 19042, 19043, 19044, 22000, 22621
- Device Type: hp notebook, dell latidute, vmware
However, I have successfully used the current version of the script on machines with the same specifications.

**Expected behavior**
The uninstall process starts, and the returned status is equal to "... cmd started removal of the Falcon sensor"

**Actual behavior**
The command returns an empty status, and the Falcon uninstallation is unsuccessful.

**Environment (please complete the following information):**
 - PSFalcon: 2.2.5

**Comment by bk-cs** [comment]
Thanks for the report! Can you update your local `public\psf-sensors.ps1` file with this one, reload PowerShell, and retry your test?

```
(Invoke-WebRequest https://raw.githubusercontent.com/CrowdStrike/psfalcon/2.2.6/public/psf-sensors.ps1).Content > $HOME\Documents\WindowsPowerShell\Modules\PSFalcon\2.2.5\public\psf-sensors.ps1
```

**Comment by dvd00** [comment]
Sorry, I'm unable to test your fix as I already uninstalled the agent after implementing a similar one. Based on the cases I've observed, it should work fine.
Thank you for the prompt reply.

**Comment by bk-cs** [comment]
Resolved in 2.2.6 release which has been published to GitHub and will be published on the PowerShell Gallery soon.
